### DISCUSSION TITLE
Zoom speed

### DISCUSSION THREAD

**Discussion by tfeldmethwalter** [google/model-viewer#4327] [Q&A]
Hello all,

I have a model (glb). This consists of many, even small, elements, which makes me zoom in relatively far.
Now the problem is that the speed of the zoom (on Windows) seems to depend on the set number of pages when I turn the scroll wheel.

![anzahl-geblaetterte-seiten](https://github.com/google/model-viewer/assets/137861794/bf4a7c75-51b9-4763-b4de-020d88d4a079)

The higher the number the faster the zooming. Is there any way to regulate this?
Deltay of the mouse wheel can unfortunately not be adjusted

Thanks and greetings

**Comment by tfeldmethwalter** [comment]
It's not very pretty, but I've found a workable solution for controlling the zoom.
If someone has a nicer solution, I am very interested in it.

![model-viewer-hack-for-zoom-1](https://github.com/google/model-viewer/assets/137861794/08eacdcf-c259-4136-bb1c-ec5efd1a558f)

![model-viewer-hack-for-zoom](https://github.com/google/model-viewer/assets/137861794/c8312990-fea7-4ea0-b941-b8a40fd36e0d)

**Comment by diegoteran** [comment]
Hi @tfeldmethwalter, 

I could see two options using our public API.
1) Similar to what you do here (using the wheel input) and then using the zoom function: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-methods-zoom
2) Increasing input sensitivity (https://modelviewer.dev/docs/#entrydocs-stagingandcameras-properties-inputSensitivity), but decreasing orbit sensitivity (https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-orbitSensitivity) so only the zoom changes behavior.

**Reply by tfeldmethwalter** [reply]
Hi @diegoteran,

thanks for the answer. I will check the effects of this adjustment once.
However, I can not currently imagine that the problem is fixed with it, since that in the script from Google uses the deltaY value to calculate. This value is, at least on Windows, variable adjustable by the user.
